.env.example

```
@@ -39,6 +39,9 @@ NEXT_PUBLIC_HARMONIA_WEB_URL="http://127.0.0.1:3001"
 # Dashboard app URL
 NEXT_PUBLIC_HARMONIA_DASHBOARD_URL="http://127.0.0.1:3003"
 
+# Admin app URL (internal tool, port 3004)
+NEXT_PUBLIC_HARMONIA_ADMIN_URL="http://127.0.0.1:3004"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)